4.7.1-windowsservercore-ltsc2016/build/Dockerfile

```diff
@@ -3,30 +3,34 @@ FROM microsoft/dotnet-framework:4.7.1-windowsservercore-ltsc2016
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 # Install NuGet CLI
+ENV NUGET_VERSION 4.4.1
 RUN New-Item -Type Directory $Env:ProgramFiles\NuGet; \
-    Invoke-WebRequest -UseBasicParsing https://dist.nuget.org/win-x86-commandline/v4.4.1/nuget.exe -OutFile $Env:ProgramFiles\NuGet\nuget.exe
+    Invoke-WebRequest -UseBasicParsing https://dist.nuget.org/win-x86-commandline/v$Env:NUGET_VERSION/nuget.exe -OutFile $Env:ProgramFiles\NuGet\nuget.exe
 
 # Install VS Test Agent
 RUN Invoke-WebRequest -UseBasicParsing https://download.visualstudio.microsoft.com/download/pr/100232757/8a386d27295953ee79281fd1f1832e2d/vs_TestAgent.exe -OutFile vs_TestAgent.exe; \
     Start-Process vs_TestAgent.exe -ArgumentList '--quiet', '--norestart', '--nocache' -NoNewWindow -Wait; \
     Remove-Item -Force vs_TestAgent.exe; \
 # Install VS Build Tools
```

Comment:
Nit - a new line before the comment looks cleaner. Same with the other file.

Reply:
You can't have a new line because the Dockerfile parser doesn't support it.  You would need the line a continuation `\`.  Because of that I am on the fence on whether or not this really adds value.  Leaving as is.